server/src/coqtop/CoqTop8.ts

```diff
@@ -250,17 +250,16 @@ export class CoqTop extends IdeSlave8 implements coqtop.CoqTop {
         ...this.settings.args
         ];
```

Comment:
I haven't ported this code to `const`, but it should be doable the same way. (But I prefer to avoid mixing refactorings with changes....).

Reply:
Done in last commit (untested yet).